src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -5,12 +5,7 @@
 import io.dropwizard.auth.Auth;
 import org.broadinstitute.consent.http.enumeration.DarStatus;
 import org.broadinstitute.consent.http.enumeration.UserRoles;
-import org.broadinstitute.consent.http.models.AuthUser;
-import org.broadinstitute.consent.http.models.DarCollection;
-import org.broadinstitute.consent.http.models.DataAccessRequest;
-import org.broadinstitute.consent.http.models.PaginationResponse;
-import org.broadinstitute.consent.http.models.PaginationToken;
-import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.*;
```

Comment:
We don't use star imports. Look for the setting in intellij that lets you set it to a very large number (i.e, 9999).
<img width="1372" alt="Screen Shot 2022-02-07 at 1 18 27 PM" src="https://user-images.githubusercontent.com/116679/152847885-bf034d5c-cf99-4e2a-9ec0-d8a00fd321a0.png">